config/projects-prod/ntap-gdp5-project.yaml

```diff
@@ -0,0 +1,23 @@
+template_path: tower-project.yaml
+stack_name: ntap-gdp5-project
+dependencies:
+  - common/nextflow-forge-iam-policy.yaml
+  - common/nextflow-launch-iam-policy.yaml
+
+parameters:
+  S3ReadWriteAccessArns:
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/bruno.grande@sagebase.org
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/jineta.banerjee@sagebase.org
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/robert.allaway@sagebase.org
+  AccountAdminArns:
+    - '{{stack_group_config.sso_admin_role.arn}}'
+    - !stack_output_external workflows-nextflow-ci-service-account::ServiceRoleArn
+  TemplateRootUrl: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com'
+  TowerForgePolicyArn: !stack_output_external nextflow-forge-iam-policy::NextFlowForgePolicyArn
+  TowerLaunchPolicyArn: !stack_output_external nextflow-launch-iam-policy::NextFlowLaunchPolicyArn
+
+stack_tags:
+  Department: SCCE
+  Project: Neurofibromatosis
+  OwnerEmail: bruno.grande@sagebase.org
+  CostCenter: NO PROGRAM / 000000  # Will be updated once a program code is available
```

Comment:
No program code?

Reply:
The contract hasn't been finalized yet, so Finance hasn't minted a program code yet.

Reply:
Interestingly, there does appear to be a code from finance (301101) for this project. I think this is because while it's not finalized, it's backdated to January, so we have it in the system already.

Reply:
(sorry for not commenting _before_ this was merged...I lost track of this PR)